ci/logging_tests_ctlplane.yml

```diff
@@ -0,0 +1,98 @@
+- name: Prepare Logging Tests
+  hosts: controller
+  gather_facts: false
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+
+- name: "RHOSO-12753"
+# description: Verify ctlplane node logging containers
+  hosts: computes
+  gather_facts: no
+  vars:
+    container_out_file: "verify_ctlplane_node_logging_containers_{{ inventory_hostname }}_lresults.log"
```

Comment:
Instead of passing the outfile name to the role, you could create an output dir with the hostname in the name e.g. ``compute-0-logging-results/`` and and pass the outfile_dir into the role, which would then create the outfiles in the outfile_dir  (this change would be in the qe_common role).

Reply:
removed all output file generation.